docs/notes/todo.md

```diff
@@ -0,0 +1,45 @@
+# Todo
+
+
+## Critical Path Tasks
+
+- [ ] Test and debug Registry routines
+- [ ] Implement file cache
+- [ ] Implement paging
+
+
+## These Tasks Would Be Nice
+
+- [ ] Separate HAL into KM DLL
+
+
+## Function Groups Totally or Partially Unimplemented
+
+- [ ] Dma functions (see hal/x86/dma.c)
+- [ ] Shutdown support (see ex/power.c)
+- [ ] Zw(Set/Get)SystemInformation (see ex/sysinfo.c)
+- [ ] Io cancelation support (see io/cancel, et al)
+- [ ] Directory change notification (see io/dir.c)
+  - [ ] Error logging (see io/errlog.c)
+  - [ ] Io completion ports (see io/iocomp.c)
+  - [ ] File locking (see io/lock.c)
+  - [ ] Hardware resource management (see io/resource.c)
+  - [ ] Exception support (see ke/catch.c)
+  - [ ] Mutex support (see nt/mutex.c)
+
+
+## Verify Implementation
+
+### Major Areas
+
+- [ ] Security support (see se/*.c)
```

Comment:
I'm the current de-facto maintainer of the security kernel subsystem, what do you want to know about it?

Reply:
Note that this is text adaptation of the existing todo file we currently have in master:
https://git.reactos.org/?p=reactos.git;a=blob;f=media/doc/todo;h=304a3adf3ffcb677d60a6b8987d18c36b32827ca;hb=HEAD#l29

Reply:
Ok that this is a text adaptation of the existing one but it doesn't answer my question -- **what there needs to be verified**? The security subsystem is in a decent shape if not perfect and the base skeleton implementation is already there (access tokens, the SRM, privilege/SID management, et al). What's left are bugfixes where necessary (just like with any other module of the ROS project, duh).

The only missing major feature is the security kernel auditing but that is a low priority as I have other things to do which are more important than this. NT6+ features can be implemented on top of the security subsystem without the need of a total rewrite.

Since @cbialorucki is touching that file, I would advise that it has to be updated reflecting the current situation of the project and plan future TODOs.